gateways/authorizenet/types.go

```diff
@@ -21,6 +21,10 @@ const (
 	ResponseCodeHeld     ResponseCode = "4"
 )
 
+const (
```

Comment:
We can probably expand this as time goes on, but right now, this is to fix the one issue mentioned in this ticket 
https://app.asana.com/0/1201437212753355/1201868442243928